dallinger/command_line/prolific.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import click
+from tabulate import tabulate
+
+logger = logging.getLogger(__name__)
+
+
+# Prolific-specific commands
+@click.group("prolific")
+@click.pass_context
+def prolific(ctx):
+    """Sub-commands for Prolific"""
+    pass
+
+
+@prolific.group("list")
+@click.pass_context
+def list(ctx):
+    """Sub-commands for listing Prolific entities"""
+    pass
+
+
+@list.command("workspaces")
+@click.option(
+    "--show-all-columns",
+    is_flag=True,
+    flag_value=True,
+    help="Display all workspace columns returned from the Prolific API",
+)
+@click.pass_context
+def list__workspaces(ctx, show_all_columns):
```

Comment:
Too many underscores?

Reply:
I did this to be in line with EC2 `list__instances`.